src/workerd/api/url.h

```diff
@@ -161,10 +161,9 @@ class URLSearchParams: public jsg::Object {
                 IteratorState,
                 valueIteratorNext)
 
-  void forEach(
-      jsg::Lock& js,
-      jsg::V8Ref<v8::Function> callback,
-      jsg::Optional<jsg::Value> thisArg);
+  void forEach(jsg::Lock&,
+               jsg::Function<void(kj::StringPtr, kj::StringPtr, jsg::Ref<URLSearchParams>)>,
+               jsg::Optional<jsg::Value>);
```

Comment:
Possibly just learned something about c++, just want to verify before I make any assumptions. Can you always just list types, and then let the implementation name the variables or is this a special case?

Reply:
Yep. I know there are differences of opinion on this about whether it is desirable but personally I prefer the more abbreviated form in declarations when the intent is clear.